### PR TITLE
add first pytest unit tests

### PR DESCRIPTION
**Pull request by tomsilver**
also make bathing and dressing environments easier to create

### CODE REVIEW

File: pyrcareworld/pyrcareworld/envs/dressing_env.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""Dressing environment for the PhyRC competition."""
+
+from pyrcareworld.envs.base_env import RCareWorld
+from pathlib import Path
+
+
+_TEMPLATE_PATH = Path(__file__).parents[3] / "template"
```
Comment (YoruCathy) [inline pyrcareworld/pyrcareworld/envs/dressing_env.py:7]: For these path, should we consider using some more configurable formats?

Something like:
```
from pathlib import Path
from pyrcareworld.envs.base_env import RCareWorld

class DressingEnv(RCareWorld):
    """Dressing environment for the PhyRC competition."""

    def __init__(self, template_path=None, executable_path=None, *args, **kwargs):
        # Allow paths to be passed as arguments or use defaults
        template_path = Path(template_path or Path(__file__).parents[3] / "template")
        executable_path = Path(executable_path or template_path / "Dressing" / "DressingPlayer.x86_64")
        
        super().__init__(executable_file=str(executable_path), *args, **kwargs)
```

File: .github/workflows/ci.yml
Comment (YoruCathy) [inline .github/workflows/ci.yml:1]: Should we cache the conda env?

**Review by YoruCathy — APPROVED** [review]
I think the current version is good to merge but added some cmments regarding making the path more configurable and adding cache for the conda env to make the workflow faster.

File: .github/workflows/ci.yml
Comment (tomsilver) [reply .github/workflows/ci.yml:1]: This is a good idea, but I couldn't figure out exactly the right way to do this after looking at it for a bit, so I'll save this for a future PR!

**Pull request merged**